docs/apv.md

```diff
@@ -291,11 +291,12 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
     * Overheids voorwerpen
     * Onder dwang iemand zijn huis betreden en dan leegroven
 3. Voertuigen kunnen ook niet meer gebruikt worden bij illegale deals en kunnen enkel met legaal geld gekocht worden.
-4. Met criminele activiteiten zoals vermeld in lid 1 worden dingen met betrekking tot drugs, overvallen, witwassen, het dragen van een masker, kogelwerend vest, holster etc bedoeld, en dus geen overtredingen.
-5. Bij een overtreding van de wet in lid 1 en/of lid 2 wordt de persoon gestraft volgens de 4e categorie en wordt het geld teruggegeven aan het slachtoffer. Wanneer de persoon het geld al heeft uitgegeven zal het resulteren in straf van de 8e categorie.
-6. Hou hiermee wel rekening dat het bij het vergrijp zoals beschreven in lid 1 niet de bedoeling is hiermee misbruik te maken van onwetendheid van mensen, dus vermijd het oplichten van nieuwe spelers.
-7. Het is voor criminelen toegestaan om mensen gedurende langere tijd te ondervragen en/of te martelen. Echter mag dit maximaal circa 2 uur duren. Er dient ook goede roleplay aan gekoppeld te worden. Dit is ter beoordeling aan een stafflid.
-8. Bij een overtreding van de regel in lid 6 wordt de persoon gestraft volgens de 1e categorie en zal het scenario onmiddellijk worden teruggedraaid.
+4. Het is de verantwoordelijkheid van de aankoper van een voertuig om te controleren welke tunes er daadwerkelijk op het voertuig zitten.
```

Comment:
Het is de verantwoordelijkheid van de kopende partij om te controleren of de aangeboden aanpassingen op het voertuig aanwezig zijn ten tijde van overdracht van het voertuig.